src/common/interfaces/event.interface.ts

```diff
@@ -122,7 +122,7 @@ export interface DsyncGroupUserAddedEventResponse extends EventResponseBase {
   data: {
     directory_id: string;
     user: DirectoryUserResponse;
-    group: Pick<EventDirectoryGroupResponse, 'id' | 'name'>;
+    group: EventDirectoryGroupResponse;
```

Comment:
Is `EventDirectoryGroupResponse` the same as `DirectoryGroupResponse` at this point?

Reply:
Sure is! Removed the event version in favor of DirectoryGroupResponse.